samples/artifacts/EventPublishing/PublishNatsinTextFormat.siddhi

```diff
@@ -0,0 +1,58 @@
+@App:name("PublishNatsinTextFormat")
+@App:description('Publish events via NATS transport in text format with default mapping and view the output on the NATS stream console.')
+
+/*
+Purpose:
+    This application demonstrates how to configure siddhi applications to publish events to the LowProductionAlertStream via NATS transport in text default format.
+
+Prerequisites:
+    1)Install the NATS Streaming server from "https://nats-io.github.io/docs/nats_streaming/gettingstarted/install.html"
+    2)The Siddhi app runtime will need the following JARs to connect to the NATS streaming server. Download those JARs using the following links and add those JARs to "{SiddhiHome}/jars" directory.
+        a)Java NATS streaming :-"https://mvnrepository.com/artifact/io.nats/java-nats-streaming/2.1.2"
+        b)JNATS:-"https://mvnrepository.com/artifact/io.nats/jnats/2.3.0"
+    3)It also needs the Protobuf bundle. Download the Protobuf bundle using the following link and add the bundle to "{SiddhiHome}/bundles" directory.
+        a)Protobuf:-"https://mvnrepository.com/artifact/com.google.protobuf/protobuf-java/3.6.1"
+        
+Configurations:
+      For local deployments to testing this sample, we need to set up these environmental variables as follows.
+        a)STAN_CLUSTER_ID="test-cluster"
```

Comment:
Same as above env setting.

Reply:
9f63596947b84bc1942360a9da56f478c9c1389b This commit fixed the above correction